corehq/apps/reminders/util.py

```diff
@@ -95,12 +95,6 @@ def get_form_name(form_unique_id):
     return app.name + "/" + module_name + "/" + form_name
 
 def get_recipient_name(recipient, include_desc=True):
-    # Circular imports
-    from corehq.apps.users.models import CouchUser
-    from corehq.apps.groups.models import Group
-    from casexml.apps.case.models import CommCareCase
-    from casexml.apps.case.models import CommCareCaseGroup
```

Comment:
found this reference still pointing to the old location of CommCareCaseGroup, and then realized that none of these imports have a circular dependency anymore